src/test/java/org/checkerframework/specimin/UnsolvedMethodInExistingClassTest.java

```diff
@@ -0,0 +1,19 @@
+package org.checkerframework.specimin;
+
+import java.io.IOException;
+import org.junit.Test;
+
+/**
+ * This test checks if Specimin can properly update the list of target files to be solved by
+ * UnsolvedSymbolVisitor in case there is an unsolved method from a class that is not added as a
+ * target file.
+ */
+public class UnsolvedMethodInExistingClassTest {
+  @Test
+  public void runTest() throws IOException {
+    SpeciminTestExecutor.runTestWithoutJarPaths(
+        "unsolvedmethodsinexistingclass",
```

Comment:
Why is the name of the directory different than the name of the class? (There is an extra `s` after `method` in the directory name.)